pkg/test/nodeclass.go

```diff
@@ -38,6 +38,13 @@ func EC2NodeClass(overrides ...v1beta1.EC2NodeClass) *v1beta1.EC2NodeClass {
 	if options.Spec.AMIFamily == nil {
 		options.Spec.AMIFamily = &v1beta1.AMIFamilyAL2
 	}
+	if options.Spec.AMISelectorTerms == nil {
+		options.Spec.AMISelectorTerms = append(options.Spec.AMISelectorTerms, v1beta1.AMISelectorTerm{
+			EKSOptimized: &v1beta1.EKSOptimized{
+				Family: v1beta1.AMIFamilyAL2,
```

Comment:
(consider for a follow-on) Should we update this from AL2 to AL2023 to match our E2E testing

Reply:
Yep, still probably good to be consistent. Can we backlog this and have it be something small that gets picked up?